doc/content/examples/index.md

```diff
@@ -17,5 +17,7 @@ A list of example simulations using MASTODON is provided below. These examples i
 - [Example 7](examples/example7.md): Demonstration of domain reduction method.
 - [Example 8](examples/example8.md): Seismic risk assessment of a generic nuclear facility using the MASTODON FTA Python module.
 - [Example 9](examples/example9.md): Soil-structure interaction analysis of an undamped rigid structure on undamped linear soil.
+- [Example 10a](examples/example10a.md): Cantilever example demonstrating central difference time integration.
+- [Example 10b](examples/example10b.md): Cantilever example demonstrating central difference time integration using HEX20 elements.
```

Comment:
This page doesn't seem to render properly on the website. There is a bunch of text in red. @mriganabh To view how the new content appears on the MASTODON website, do the following:
- Go to the `doc` folder in MASTODON
- Then run `./moosedocs.py build --serve`
- The above command should give you a https link which shows the MASTODON website with the new changes. Then, you can verify if everything appears correctly.